test/hotspot/jtreg/runtime/jni/checked/TestCheckedReleaseArrayElements.java

```diff
@@ -44,10 +44,11 @@ public static void main(String[] args) throws Throwable {
         if (args == null || args.length == 0) {
             test();
         } else {
-            OutputAnalyzer output =
-                ProcessTools.executeTestJvm("-Xcheck:jni",
-                                            "-Djava.library.path=" + Utils.TEST_NATIVE_PATH,
-                                            "TestCheckedReleaseArrayElements");
+            ProcessBuilder pb =
+                ProcessTools.createJavaProcessBuilder("-Xcheck:jni",
+                                                      "-Djava.library.path=" + Utils.TEST_NATIVE_PATH,
+                                                      "TestCheckedReleaseArrayElements");
+            OutputAnalyzer output = ProcessTools.executeProcess(pb);
```

Comment:
I would add a comment above L51:
// Uses executeProcess() instead of executeTestJvm() to avoid passing options
// that might generate output on stderr (which should be empty for this test).

Reply:
Done.